ibm_catalog.json

```diff
@@ -202,7 +198,93 @@
             {
               "key": "skip_iam_authorization_policy"
             }
-          ]
+          ],
+          "dependencies": [
+            {
+              "name": "deploy-arch-ibm-account-infra-base",
+              "catalog_id": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3",
+              "flavors": ["standard"],
+              "id": "63641cec-6093-4b4f-b7b0-98d2f4185cd6-global",
+              "ignore_auto_referencing": ["*"],
+              "input_mapping": [
+                {
+                  "dependency_output": "workload_resource_group_name",
+                  "version_input": "resource_group_name"
+                },
+                {
+                  "dependency_input": "prefix",
+                  "version_input": "prefix",
+                  "reference_version": true
+                },
+                {
+                  "value": true,
+                  "version_input": "use_existing_resource_group"
+                }
+              ],
+              "optional": true,
+              "on_by_default": true,
+              "version": "^v1.18.0"
+            },
+            {
+              "name": "testing-deploy-arch-ibm-kms",
+              "id": "85fdbd7a-8c77-4abd-b716-653a31f3aba9-global",
+              "version": "^1.0.6",
+              "flavors": ["standard"],
+              "catalog_id": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3",
+              "optional": true,
+              "on_by_default": true,
+              "ignore_auto_referencing": ["*"],
+              "input_mapping": [
```

Comment:
I see when the addon is enabled the value of `enable_kms_encryption` is false we should set it to true when the addon is enabled by doing:
```json
{
  "value": true,
  "version_input": "enable_kms_encryption"
}
```